feat(persist-state): 🔥 excludeKeys operator (#135)
ngneat/elf: docs/docs/features/persist-state.mdx

  storage: localForage,
});
```

> ## Excluding keys from saved state

Reviewer: ```suggestion
## Excluding keys from the state
```

> `excludeKeys()` operator can be used to exclude some keys from saved state:

Reviewer: ```suggestion
The `excludeKeys()` operator can be used to exclude keys from the state:
```

Author: Haven't fixed those yet. Was the merge intentional?

Reviewer: No, please fix it

Author: Should I leave the operator name as is or do you want to rename it to `omitKeys`?

Reviewer: leave it

Author: @NetanelBasal thank you very much for all the awesome open source libraries you have created and articles you have written 🙏

Reviewer: Thanks

```ts
import { excludeKeys, persistState, localStorageStrategy } from '@ngneat/elf-persist-state';

persistState(todoStore, {
  key: 'todos',
  storage: localStorageStrategy,
  source: () => todoStore.pipe(

>     excludeKeys(['ids', 'entities'])

Reviewer: I think that `omitKeys` is more suitable. See lodash naming for example https://lodash.com/docs/4.17.15#omit

Author: Just a matter of personal preference. `omit` might be better because it's shorter and sounds a bit more formal, but `exclude` is more widely used word and when you have auto-completion in your IDE the length doesn't really matter.
Anyway, you can choose whichever you prefer.
